### PR TITLE
[spec] Fix signature of `table.grow` in instructions index

### PR DESCRIPTION
**Pull request by fitzgen**
It pushes the old table size on success or `-1` on failure.

@rossberg, please take a look.

### CODE REVIEW

**Comment by fitzgen** [comment]
Just went and linked my github account with my w3c account. Not sure how to retrigger the check.

File: document/core/appendix/index-instructions.rst
Comment (rossberg) [inline document/core/appendix/index-instructions.rst:279]:
```suggestion
:math:`\TABLEGROW~x`                       :math:`\hex{FC}~\hex{0F}`  :math:`[t~\I32] \to [\I32]`                     :ref:`validation <valid-table.grow>`     :ref:`execution <exec-table.grow>`                             
```

**Review by rossberg — APPROVED** [review]
Thanks! LGTM modulo missing backslash.

**Review by rossberg — CHANGES_REQUESTED** [review]
Oh, sorry, actually, this is a generated file. You'll need to change gen-index-instructions.py and then rerun it.

**Comment by fitzgen** [comment]
Updated the script and re-ran it.

**Review by rossberg — APPROVED** [review]
Thanks!

**Comment by fitzgen** [comment]
Thanks for review!

Note: I don't have commit access to this repo, so I'll need your help actually landing this.

**Comment by rossberg** [comment]
@fitzgen, IPR check seems to fail, have you registered with the CG under the same email?

**Comment by fitzgen** [comment]
Yes, I've connected my github account, but I did it after opening this PR. Not sure how to re-run that check.

**Comment by rossberg** [comment]
(I'm overriding IPR check since the fix is trivial.)

**Pull request merged**
